mlir/lib/Driver/Pipelines.cpp

```diff
@@ -0,0 +1,204 @@
+// Copyright 2024 Xanadu Quantum Technologies Inc.
```

Comment:
The pipeline defined here will only act as a default pipeline for the executable if no manual pipeline or pass is specified. Otherwise it gets overridden. The default pipeline in the frontend should be removed at the end after all the frontend changes are done which comes in the next PRs. Right now, the frontend does not use the executable so the default pipeline defined in the python frontend is used.

Reply:
> The default pipeline in the frontend should be removed at the end after all the frontend changes are done which comes in the next PRs. Right now, the frontend does not use the executable so the default pipeline defined in the python frontend is used.

I see, thanks for the context!